plugins/jobs/src/js/JobDelete.js

```diff
@@ -0,0 +1,92 @@
+import * as React from "react";
+import { componentFromStream } from "data-service";
+import PropTypes from "prop-types";
+import { deleteJob } from "#SRC/js/events/MetronomeClient";
+import { Subject } from "rxjs/Subject";
+import "rxjs/add/operator/map";
+import "rxjs/add/operator/switchMap";
+import "rxjs/add/operator/do";
+import "rxjs/add/operator/combineLatest";
+import "rxjs/add/operator/startWith";
+import "rxjs/add/operator/catch";
+
+import JobDeleteModal from "./components/JobDeleteModal";
+
+function isTaskCurrentRunning(errorMessage) {
+  return /stopCurrentJobRuns=true/.test(errorMessage);
```

Comment:
🎨 why did you went with a regex for such a simple match? I think it is _way_ faster to just use `String.includes` 🤔

Reply:
ah, I did comment on it :D